packages/api-explorer/src/lib/filter-empty-formdata.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2019-present Mia s.r.l.
+ * All rights reserved
+ */
+
+const isSchemaValid = schema => schema && Object.keys(schema).length > 0
```

Comment:
A schema in order to be valid needs only to have > 0 keys?

Reply:
A better/more explicit function name could be `isSchemaUsable`.